Meshtastic/Views/Messages/MessageText.swift

```diff
@@ -170,3 +124,79 @@ private extension MessageDestination {
 		}
 	}
 }
+
```

Comment:
The app uses the themes and the blue is an accent color which the user can change on MacOS, changing the color to an explicit blue breaks this pattern.